frontend/packages/core/src/NPS/banner.tsx

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+
+import type { EmojiType } from "../Assets/emojis";
+import { HappyEmoji } from "../Assets/emojis";
+import { Button } from "../button";
+import { useAppContext } from "../Contexts";
+import { FeatureOn, SimpleFeatureFlag } from "../flags";
+import Grid from "../grid";
+import Paper from "../paper";
+import styled from "../styled";
+import { Typography } from "../typography";
+
+export interface BannerFeedbackProps {
+  /**
+   * Whether this component should appear integrated into the page versus in its own container
+   * @defaultValue false
+   */
+  integrated?: boolean;
+  /**
+   * The icon to display or null if none
+   * @defaultValue <HappyEmoji />
+   */
+  icon?: EmojiType | React.ReactNode | null;
+  /**
+   * Feedback text
+   * @defaultValue "Enjoying this feature? We would like your feedback!"
+   */
+  feedbackText?: string | null;
+  /**
+   * Feedback Button Text
+   * @defaultValue "Give Feedback"
+   */
+  feedbackButtonText?: string;
+  /**
+   * Default option in dropdown to select
+   * @defaultValue ""
+   */
+  defaultOption?: string;
+}
+
+const StyledPaper = styled(Paper)({
+  padding: "0px 16px",
+});
+
+/**
+ * An NPS Feedback Banner which will open up the NPSHeader component to ask for feedback
+ */
+export const Banner = ({
+  integrated = false,
+  icon = <HappyEmoji />,
+  feedbackText = "Enjoying this feature? We would like your feedback!",
+  feedbackButtonText = "Give Feedback",
+  defaultOption,
+}: BannerFeedbackProps) => {
+  const { triggerHeaderItem } = useAppContext();
+  const banner = (
+    <Grid container direction="row" spacing={1} alignItems="center">
```

Comment:
really minor but wdyt about center justifying the whole thing? On larger screens I feel like there could be a ton of white space on the right with the current left justification

Reply:
I think it looks pretty good, I'll modify it

![Screenshot 2023-01-09 at 2 26 08 PM](https://user-images.githubusercontent.com/8338893/211420702-281e65a4-74c2-4573-a0c7-1d26efc37f48.png)